wsgi.py

```diff
@@ -11,7 +12,7 @@
 """
 
 application.register_providers(
-    Kernel,
+    TestsKernel,
```

Comment:
Here `Kernel` was the base Kernel used to define the minimal specifications for the framework, and `ApplicationKernel` used to define project-specific  and feature configuration. I think we should not removed this.

If you want to split the tests part maybe we should move it to its own `TestsKernel`, without removing `Kernel` and then `TestsKernel` would contain only the `register_testing()` method..

But I am not sure about what you want to split this and why you want to remove the `tests/` folder ?

Reply:
> Here `Kernel` was the base Kernel used to define the minimal specifications for the framework, and `ApplicationKernel` used to define project-specific and feature configuration. I think we should not remove this.
> 

Hi @girardinsamuel 

This is correct, but the problem with this minimum  config  is that it incorporates the tests into the main runtime even if not doing tests.
```python
from ..tests.HttpTestResponse import HttpTestResponse
from ..tests.TestResponseCapsule import TestResponseCapsule
```
This means that you can never slim down the package (for production deployment) by removing the tests folder.
Currently I use this method (of stripping tests and other unused files from the package to create pre-compiled AWS Lambda Layers.  But I cannot strip the tests from Masonite due to their integration point in the Kernel
I hope this clarifies the usage scenario for you

> If you want to split the tests part maybe we should move it to its own TestsKernel, without removing Kernel and then TestsKernel would contain only the register_testing() method..
>
Yeah that makes sense. I was just trying to keep the same functionality but if we dont need all that for the tests then thats much cleaner.
 ... Let me try this and make sure I can get it working

Reply:
> Here `Kernel` was the base Kernel used to define the minimal specifications for the framework, and `ApplicationKernel` used to define project-specific and feature configuration. I think we should not remove this.
> 
The point of altering this file was that it is ONLY used for tests
ie the basepath is set to the `tests` folder and `ApplicationKernel` ia imported from `tests` too 
```python
from tests.integrations.config.providers import PROVIDERS
from tests.integrations.app.Kernel import Kernel as ApplicationKernel

# here the project base path is tests/integrations
application = Application(base_path("tests/integrations"))
```
Thia file would not obviously be used when building a starting app